Sources/Foundation/NSNumber.swift

```diff
@@ -9,6 +9,7 @@
 
 
 @_implementationOnly import _CoreFoundation
+@_spi(SwiftCorelibsFoundation) @_exported import FoundationEssentials
```

Comment:
Is it `@_exported` so that  `_typeByName("Foundation._FoundationNSNumberInitializer")` is visible to FoundationEssentials?

Reply:
The `@_exported` attribute is the attribute that re-exports all of the symbols from `FoundationEssentials`. All of our imports of `FoundationEssentials` and `FoundationInternationalization` are `@_exported` so that `Foundation.URL == FoundationEssentials.URL` because `Foundation` is re-exporting `FoundationEssentials` and everything within it, so it's not specific to this change it's just something we have to add to every import of those modules